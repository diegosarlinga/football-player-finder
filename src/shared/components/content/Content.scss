.Content-header {
  padding: 15px 0;
  margin: 0;
}

.Content-body {
  padding: 20px 0;
}