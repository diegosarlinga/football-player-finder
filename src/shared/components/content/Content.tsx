import React from 'react';
import './Content.scss';

interface ContentProps {
  title: string;
  className?: string;
}
const Content: React.FC<ContentProps> = (props) => {
  return (
    <div className={props.className}>
      <h2 className="Content-header text-center bg-primary text-white">
        {props.title}
      </h2>
      <div className="content container-fluid Content-body">
        {props.children}
      </div>
    </div>
  );
}

export default Content;
