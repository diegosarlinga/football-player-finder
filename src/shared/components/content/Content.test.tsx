import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Content from './Content';

describe('Content component tests', () => {
  let wrapper: ShallowWrapper;

  describe('when rendered', () => {
    beforeEach(() => {
      const props = {
        title: 'Content title'
      };
      wrapper = shallow(<Content {...props}/>);
    });
  
    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });

  });

});
