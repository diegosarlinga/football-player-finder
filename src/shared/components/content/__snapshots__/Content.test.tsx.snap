// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component tests when rendered should match snapshot 1`] = `
<div>
  <h2
    className="Content-header text-center bg-primary text-white"
  >
    Content title
  </h2>
  <div
    className="content container-fluid Content-body"
  />
</div>
`;
